### ISSUE TITLE
Missing Test Case - 808. Soup Servings

### ISSUE THREAD

**Issue by devansh-agarwal121** [LeetCode-Feedback/LeetCode-Feedback#15137]
#### Your LeetCode username
devansh_agarwal

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [x] Missing Test Cases 

#### Description of the bug

My code is accepted but gives wrong answer for multiple custom test cases

#### Code you used for Submit/Run operation

```
class Solution {
public:
    double solve (double A, double B, vector < pair <int, int> >& options, vector < vector <double> >& dp) {
        if (A <= 0  &&  B <= 0) return 0.5;
        if (A <= 0) return 1.0;
        if (B <= 0) return 0.0;

        if (dp[A][B] != -1) return dp[A][B];

        double prob = 0.0;
        for (auto& serving : options) {
            int A_taken = serving.first;
            int B_taken = serving.second;
            prob += solve(A - A_taken, B - B_taken, options, dp);
        }

        return dp[A][B] = 0.25 * prob;
    }
    double soupServings(int n) {
        if (n > 3275) return 1;
        vector < pair <int, int> > options = {{100, 0}, {75, 25}, {50, 50}, {25, 75}};
        vector < vector <double> > dp (n + 1, vector <double> (n + 1, -1.0));
        return solve(n, n, options, dp);
    }
};
```

#### Language used for code

C++

#### Expected behavior

I expected my code to not be accepted as I tested for custom value "n = 4275" which is the last value of n for which my code gave wrong answer. But since the testcases are not set upto this value, it got accepted,

#### Screenshots

<img width="528" alt="Screenshot 2023-07-29 at 7 27 26 PM" src="https://github.com/LeetCode-Feedback/LeetCode-Feedback/assets/111650966/b9c04ac1-550d-4008-aaf6-1577a4348dfe">
<img width="532" alt="Screenshot 2023-07-29 at 7 27 34 PM" src="https://github.com/LeetCode-Feedback/LeetCode-Feedback/assets/111650966/bbc4c658-3e07-42cb-bdbd-ff0da5edc9b5">

#### Additional context

In summary, you can add the testcase "n = 4275" having expected answer = "0.99998"

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.
